packages/contracts-hyperlane/src/registry.ts

```diff
@@ -0,0 +1,128 @@
+import { GithubRegistry, IRegistry } from "@hyperlane-xyz/registry";
+import { Address } from "viem";
+
+type MaybePromise<T> = T | Promise<T> | PromiseLike<T>;
+
+/** Hyperlane Registry Chain Metadata (sdk type inference seems to break) */
+export interface HyperlaneChainMetadata {
```

Comment:
Where was it breaking?

Reply:
Need to import for inference and want to avoid extra dependency. Fine for now.